### PR TITLE
Update README to define Codegen enum best practice

### PR DESCRIPTION
**Pull request by glureau-betclic**
https://github.com/square/moshi/issues/1147

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -557,6 +557,16 @@ data class BlackjackHand(
 )
 ```
 
+or for the enum classes:
```
Comment (ZacSweers) [inline README.md:560]: Can you add more here? This is vague on its own and should actually go on the proguard rules section

Comment (glureau-betclic) [reply README.md:560]: I'm afraid I don't have enough knowledge to be sure of what it changes compared to a simple @Keep, I had the feeling it was more than just proguard.

Comment (ZacSweers) [reply README.md:560]: I mean just make this a full sentence and put it in the correct proguard section of the readme

**Comment by ZacSweers** [comment]
Closing for now as this has gone stale

**Pull request closed without merging**